mooncake-store/include/rpc_service.h

```diff
@@ -255,6 +259,30 @@ class WrappedMasterService {
         return response;
     }
 
+    RemoveAllResponse RemoveAll() {
+        ScopedVLogTimer timer(1, "RemoveAll");
+        timer.LogRequest("action=remove_all_objects");
+
+        // Increment request metric
+        MasterMetricManager::instance().inc_remove_all_requests();
+
+        RemoveAllResponse response;
+        response.error_code = master_service_.RemoveAll();
+
+        // Track failures if needed
+        if (response.error_code != ErrorCode::OK) {
+            MasterMetricManager::instance().inc_remove_all_failures();
+            LOG(ERROR) << "RemoveAll failed with error code: "
+                      << static_cast<int>(response.error_code);
+        } else {
+            // Reset related metrics on successful remove all
+            MasterMetricManager::instance().reset_key_count();
+            MasterMetricManager::instance().reset_allocated_size();
```

Comment:
I'm not sure whether this is safe, as a concurrent put operation might occur during removeAll, potentially leading to inconsistent metrics.

Reply:
The RemoveAll operation seldom occurs. Perhaps we should add a read/write lock for this operation?